react-with-graphql-tutorial.md

```diff
@@ -397,9 +400,9 @@ query OrganizationForLearningReact {
 }
 {{< /highlight >}}
 
-As you can see, a `first` argument is passed to the `repositories` list field. That's how you can specify how many items from the list you expect in the result. Furthermore, the query shape doesn't need to follow the `edges` and `node` structure, but it's one of a few solutions to define paginated data structures and lists with GraphQL. Actually it follows the interface description of Facebook's GraphQL client called Relay. GitHub followed this approach and adopted it for their own GraphQL pagination API. Later, you will learn in the exercises about other strategies to implement pagination with GraphQL.
+A`first` argument is passed to the `repositories` list field that specifies how many items from the list are expected in the result. The query shape doesn't need to follow the `edges` and `node` structure, but it's one of a few solutions to define paginated data structures and lists with GraphQL. Actually, it follows the interface description of Facebook's GraphQL client called Relay. GitHub followed this approach and adopted it for their own GraphQL pagination API. Later, you will learn in the exercises about other strategies to implement pagination with GraphQL.
```

Comment:
"A`first` argument is passed to ..." to "A `first` argument is passed to ..."

Reply:
A `first` argument is passed to the `repositories` list field that specifies how many items from the list are expected in the result. The query shape for requesting paginated data doesn't have to have the `edges` and `node` structure, but it's one of a few solutions to define paginated data structures and lists with GraphQL. Actually, it follows the interface description of Facebook's GraphQL client called Relay. GitHub followed this approach and adopted it for their own GraphQL pagination API. Later, you will learn in the exercises about other strategies to implement pagination with GraphQL.